.ci/ci_cache_components.sh

```diff
@@ -58,7 +58,7 @@ cache_kernel_artifacts() {
 # This builds cloud hypervisor
 cache_cloud_hypervisor() {
 	local cloud_hypervisor_binary="cloud-hypervisor"
-	local cloud_hypervisor_binary_path="/usr/bin"
+	local cloud_hypervisor_binary_path="${GOPATH}/src/github.com/intel/cloud-hypervisor/cloud-hypervisor/"
```

Comment:
woot!  I thought the path was different https://github.com/cloud-hypervisor/cloud-hypervisor

Reply:
hmm, I thought your one was the current one @devimc /cc @sameo to confirm which URL is current...

Reply:
@devimc it redirects to the same page...I am using the one that is being used for packaging

Reply:
@devimc just to clarify `github.com/intel/cloud-hypervisor/cloud-hypervisor/ ` is the place where the binary is created (inside that specific folder)

Reply:
@devimc fixed the issue with the url

Reply:
thanks